WORKSPACE

```diff
@@ -76,18 +78,19 @@ skydoc_repositories()
 # For Skylint
 # Once https://github.com/bazelbuild/bazel/issues/4086 is done, this should be
 # much simpler
+bazel_version = "0.27.0"
```

Comment:
do we need such an old version of Bazel here?

Reply:
Let me see what is the alternative for Skylint now.

Reply:
discussed offline: we need this to run Skylint because it's not included in later version; we should use Buildifier, but that's another story